externs/es6.js

```diff
@@ -2035,6 +2035,31 @@ function WeakRef(value) {}
  */
 WeakRef.prototype.deref = function() {};
 
+/**
+ * @constructor
+ * @struct
+ * @param {function(HOLDINGS)} cleanupCallback
+ * @template TARGET, HOLDINGS, TOKEN
```

Comment:
From the spec it seems like the `cleanupCallback` accepts a single `heldValue` at any time. So maybe change `HOLDINGS` to `HELDVALUE` to avoid confusion.

https://tc39.es/ecma262/#sec-finalization-registry-constructor

Reply:
Fixed.